public/app/plugins/panel/graph/series_overrides_ctrl.ts

```diff
@@ -152,6 +152,7 @@ export function SeriesOverridesCtrl($scope, $element, popoverSrv) {
   $scope.addOverrideOption('Z-index', 'zindex', [-3, -2, -1, 0, 1, 2, 3]);
   $scope.addOverrideOption('Transform', 'transform', ['negative-Y']);
   $scope.addOverrideOption('Legend', 'legend', [true, false]);
+  $scope.addOverrideOption('Tooltip', 'tooltip', [true, false]);
```

Comment:
I wonder if inverting this makes more sense and is more intutive, make the option read: `Hide in tooltip` where default is false. Boolean override options that default to false are better I think.

Reply:
It is more intuitive. I changed like `hideTooltip`.